internal/logging/logger.go

```diff
@@ -54,3 +53,26 @@ func InitializeLogger(format, filename string, stdout bool) error {
 
 	return nil
 }
+
+func setLevelStr(level string, log bool) {
+	switch level {
+	case "error":
+		logrus.SetLevel(logrus.ErrorLevel)
+	case "warn":
+		logrus.SetLevel(logrus.WarnLevel)
+	case "info":
+		logrus.SetLevel(logrus.InfoLevel)
+	case "debug":
+		logrus.SetLevel(logrus.DebugLevel)
+	case "trace":
+		logrus.SetLevel(logrus.TraceLevel)
+	default:
+		level = "info (default)"
```

Comment:
did you just forget to remove this line or do I miss something?

Reply:
This was when users didn't specify a value, and it's used for logging what level the logger is at.